src/collections/blog/2023/2023-08-23-importance-of-github-stars/index.mdx

```diff
@@ -0,0 +1,97 @@
+---
+title: The Importance of Github Stars
+subtitle: Learn why stars matter
+date: 2023-08-23 12:30:05 -0530
+author: Victoria Nduka
+thumbnail: ./Hero-Image.svg
+darkthumbnail: ./Hero-Image.svg
+category: Community
+description: "Learn why stars matter"
+tags:
+ - Community
+ - Open source
+ - Meshery
+type: Blog
+resource: false
+published: true
+---
+
+import { BlogWrapper } from "../Blog.style.js";
+
+import Before from "./Before.JPG";
+import After from "./After.JPG";
```

Comment:
The build preview is failing @nwanduka 

Looking through the logs I feel it is a problem at the path of the image. Make sure it is correct.

Reply:
Alright. I'll do that and revert to you